### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by GurmanToor**
- changed response names to be camelCase style
- **In description of resources section:** specified units for temperature, date, humidity, windspeed
- Added response status
- Changed responses for temperature, humidity and windspeed to be returned as integers rather than strings

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,25 +23,26 @@
...
 {
-    {
-        "City": cityName,
-        "StartDate": startDate,
-        "EndDate": endDate, 
-        "AverageHighTemperature": averageHighTemperature,
-        "AverageLowTemperature": averageLowTemperature,
+    "response": {
+        "city": cityName,
+        "startDate": startDate(yyyy/mm/dd),
```
Comment (nuridak) [inline README.md]: Shouldn't we put it into the double quotes like ("yyyy/mm/dd")?

**Review by nuridak — COMMENTED** [review]
Only question is about the double quotes

File: README.md (hunk 1 above)
Comment (GurmanToor) [reply README.md]: Sure, we can do that

Comment (GurmanToor) [reply README.md]: @nuridak committed the new changes

File: README.md [hunk 2]
```diff
@@ -23,25 +23,26 @@
...
 {
-    {
-        "City": cityName,
-        "StartDate": startDate,
-        "EndDate": endDate, 
-        "AverageHighTemperature": averageHighTemperature,
-        "AverageLowTemperature": averageLowTemperature,
+    "response": {
+        "city": cityName,
+        "startDate": startDate("yyyy/mm/dd"),
```
Comment (nuridak) [inline README.md:28]: I am sorry, but I did not notice it from the first review but the format is not consistent with the one from the sample response
It should be : yyyy-mm-dd

Comment (nuridak) [reply README.md:28]: I will just merge it in and quickly fix the format later

**Pull request merged**